### ISSUE TITLE
BER decoding fails during Varbind packet extraction due to noSuchInstance on some

### ISSUE THREAD

**Issue by nathanIL** [secdev/scapy#3900]
### Brief description

Any OID that has _noSuchInstance_ (as shown in in tcpdump) fails with:

```
Exception has occurred: BER_Decoding_Error
Unknown prefix [81] for [b'\x81\x00']
### Already decoded ###
None
### Remaining ###
```

this in turn causes the whole SNMPResponse packet to be dropped/ignored.

I'm parsing a pcap file with sniff(...), no filters, doing nothing special.

### Scapy version

2.5.0

### Python version

3.11

### Operating system

Linux x86_64

### How to reproduce

pick some pcap where there are noSuchIsntace responses.

Basically, run tcpdump on some device and ask/get non existing OIDs

### Actual result

The whole SNMP response packet is ignored

### Expected result

At least return the packet with the right / extracted varbinds and ignore the noSuchInstance ones, or create new relevant type NONE ASN1_Tag doesnt handle that.

**Comment by gpotter2** [comment]
Hi, could you provide a pcap? (or something reproducible?) Thanks

**Comment by nathanIL** [comment]
> Hi, could you provide a pcap? (or something reproducible?) Thanks

sure, i will generate one and attach asap
